.npmrc

```diff
@@ -4,5 +4,8 @@ allows minor upgrades for that package, but after npm config set save-prefix='~'
 ~1.2.3 which only allows patch upgrades.
 save-prefix=""
 
+# required because the braft-editor still enforces react version 16 as a peer dependency
+legacy-peer-deps=true
+
```

Comment:
This should no longer be needed since components version:
```
"@meemoo/react-components": "2.12.12",
```

Now the braft editor is correctly packaged by rollup so no requirements should be leaking out to the client npm install.

```suggestion
```

Reply:
Fixed in https://github.com/viaacode/hetarchief-client/commit/a05213db3c6d2c8bf1d24f9914baf7dddfc25dc8